qds_sdk/clusterv2.py

```diff
@@ -0,0 +1,549 @@
+from qds_sdk.qubole import Qubole
+from qds_sdk.resource import Resource
+from qds_sdk.cloud.cloud import Cloud
+from qds_sdk.engine import Engine
+from qds_sdk import util
+import argparse
+import json
+
+def str2bool(v):
+    return v.lower() in ("yes", "true", "t", "1")
+
+class ClusterCmdLine:
+
+    @staticmethod
+    def parsers(action):
+        argparser = argparse.ArgumentParser(
+            prog="qds.py cluster",
+            description="Cluster Operations for Qubole Data Service.")
+        subparsers = argparser.add_subparsers(title="Cluster operations")
+
+        create = subparsers.add_parser("create", help="Create a new cluster")
```

Comment:
Shouldn't we add subparsers based on the action type? For example we should add `create` subparser only if `action == "create"`.